src/components/LiveCursors/Cursors.tsx

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from 'react'
+import { Space, SpaceMember } from '@ably-labs/spaces'
+import CursorSvg from './CursorSvg'
+import useCursor from './use-cursor'
+
+// 💡 This component is used to render the cursor of the user
+const YourCursor = ({
+  user,
+  space,
+}: {
+  user: SpaceMember | undefined
+  space?: Space
+}) => {
+  const [cursorPosition, setCursorPosition] = useState<{
+    [key: string]: number
+  }>({ left: 100, top: 100 })
+  const handleSelfCursorMove = useCursor(setCursorPosition, space)
+  if (!user) {
+    return null
+  }
+  const {
+    gradientStartClass,
+    gradientEndClass,
+    cursorStartColor,
+    cursorEndColor,
+  } = user?.profileData.userColors
```

Comment:
if user is defined by this point we don't need the `user?` I think?

Reply:
Sorry, I missed these. Fixed now.